mac/Keyman4Mac/Keyman4Mac/AppDelegate.m

```diff
@@ -114,15 +113,14 @@ CGEventRef eventTapFunction(CGEventTapProxy proxy, CGEventType type, CGEventRef
     BOOL handled = NO;
     
     if (type == NX_KEYDOWN) {
+        NSLog(@"AppDelegate eventTapFunction key down event: %@", event);
         // Key down event
         NSEvent *mEvent = [NSEvent eventWithCGEvent:event];
         KMEngine *kme = [[KMEngine alloc] initWithKMX:kmx context:contextBuffer verboseLogging:debugMode];
-        NSArray *actions = [kme processEvent:mEvent];
-        //if (debugMode)
-            NSLog(@"%@", actions);
-        for (CoreAction *action in actions) {
-            if (action.isCharacter) {
-                NSString *output = action.content;
+        CoreKeyOutput *coreKeyOutput = [kme processEvent:mEvent];
```

Comment:
I'm curious, because this function seems to ignore backspaces entirely?

Reply:
This is the old sample app which I have tried to ignore other than making sure it still builds. However, I often confuse it for Keyman IM when it comes up in search results, so I should eventually either confirm that it is actually useful or remove it from the project.

Reply:
Ah, yes, I make the same mistake too. At the very least, let's rename it!